AutoCollection/CorrelationContextManager.ts

```diff
@@ -39,6 +41,8 @@ export interface CorrelationContext {
 export class CorrelationContextManager {
     private static enabled: boolean = false;
     private static hasEverEnabled: boolean = false;
+    private static session: any;
+    private static cls: any;
```

Comment:
You can use the import style seen in the top of `https://github.com/Microsoft/ApplicationInsights-node.js/blob/develop/AutoCollection/diagnostic-channel/initialization.ts` to get the proper types for this without requiring the cls-hooked package in the resulting JS.